h/static/scripts/site.js

```diff
@@ -20,6 +20,7 @@ const SearchBarController = require('./controllers/search-bar-controller');
 const SearchBucketController = require('./controllers/search-bucket-controller');
 const SignupFormController = require('./controllers/signup-form-controller');
 const TooltipController = require('./controllers/tooltip-controller');
+const ShareWidgetController = require('./controllers/share-widget-controller');
```

Comment:
Nit: The items here and keys in the `controllers` array below should be sorted alphabetically.